homeassistant/components/totalconnect/__init__.py

```diff
@@ -22,6 +22,7 @@
             {
                 vol.Required(CONF_USERNAME): cv.string,
                 vol.Required(CONF_PASSWORD): cv.string,
+                vol.Required(CONF_USERCODES): vol.Schema({}, extra=vol.ALLOW_EXTRA),
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
Sorry I missed that announcement.  I'll adjust accordingly